test/source/TestVector.cpp

```diff
@@ -1453,6 +1520,7 @@ int TestVector()
 	// unique_ptr tests
 	{
 		// Simple move-assignment test to prevent regressions where eastl::vector utilizes operations on T that are not necessary.
+#if EASTL_MOVE_SEMANTICS_ENABLED
```

Comment:
Ok, I have recommit this part, moving the pre-processor on the previous line in order to have the same style has the other cases.